app/shareCircle.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { SafeAreaView, Text, View, Share } from 'react-native';
+import { styled } from 'nativewind';
+import { Button } from '../components/Buttons';
+import QRCode from "react-qr-code";
+
+const StyledSafeArea = styled(SafeAreaView);
+const StyledView = styled(View);
+const StyledText = styled(Text);
+
+const shareCircle = async () => {
+  try {
+    await Share.share({
+      title: 'Hey, this message was sent from our app! https://github.com/aidan-hubley/Prayer-Circle',
+      message: 'Hey, this message was sent from our app! https://github.com/aidan-hubley/Prayer-Circle',
+      url: 'Hey, this message was sent from our app! https://github.com/aidan-hubley/Prayer-Circle'
+    });
+  } catch (error) {
+    console.error('Error sharing:', error);
+  }
+}
+
+export default function Page() {
+	return (
+		<StyledSafeArea className='bg-offblack border' style={{ flex: 1 }}>
+			<StyledView className='flex-1 items-center'>
+				<StyledText className='absolute top-10 text-3xl text-offwhite bg-grey text-center h-[60px] px-[35px] py-[12px] rounded-full'>
```

Comment:
Same issue as the other page here, where the corners dont look right. again I think it should be inside of a StyledView

Reply:
Not sure what you meant previously nor what you mean here. Corners look like the figma to me

Reply:
@aidan-hubley I will send a screenshot in the slack regarding this corner issue